client/components/BudgetGrid/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component, PropTypes } from 'react';
+
+import './style.scss';
+
+class BudgetGrid extends Component {
+  static propTypes = {
+    data: PropTypes.object
+  }
+  static defaultProps = {
+    data: {}
+  }
+
+  formatAmount = (amount) => {
+    const isNegative = amount < 0;
+    const formatValue = Math.abs(amount).toLocaleString('en-us', { style: 'currency', currency: 'USD' });
```

Comment:
Can you do me a favor and open a file called Polyfills.md. Add `Number.prototype.toLocaleString`. We'll need to add polyfills for that one